### ISSUE TITLE
Program stuck during validation process.

### ISSUE THREAD

**Issue by GauravGajbhiye** [a-nau/CubeRefine-R-CNN#2]
My code runs properly for training process (upto 1000 iteration) but stuck at validation process. I am using 4 GPUS to run the code.

Stuck after evaluating following lines:
[08/10 02:48:12] d2.evaluation.evaluator INFO: Start inference on 1 batches
[08/10 02:49:00] d2.evaluation.evaluator INFO: Inference done 1/1. Dataloading: 0.0000 s/iter. Inference: 0.1117 s/iter. Eval: 0.0061 s/iter. Total: 0.1178 s/iter. ETA=0:00:00
[08/10 02:49:06] d2.evaluation.evaluator INFO: Total inference time: 0:00:05.186749 (5.186749 s / iter per device, on 4 devices)
[08/10 02:49:06] d2.evaluation.evaluator INFO: Total inference pure compute time: 0:00:00 (0.111703 s / iter per device, on 4 devices)

Also, could you please provide output visualization and test script separately for checking the performance on my dataset.
Thank you in advance for your cooperation.

**Comment by a-nau** [comment]
Hi @GauravGajbhiye

I never experienced that problem. Which data are you using? I also only trained on 1 GPU.

I currently don't have any visualization script ready, but feel free to check out the visualizations of Mesh R-CNN and Cube R-CNN, that should help a lot.

**Comment by GauravGajbhiye** [comment]
Hi @a-nau 

I have used the dataset which you provided in this repo (10 images in training and 1 image in validation set). The code runs smoothly for training process but stuck during validation process. 

With your pretrained weights, now I can manage to test the output for my images using Cube RCNN's visualization code.

Will you please provide the method and tool to create the 3D annotations for custom image?
Thank you!

**Comment by a-nau** [comment]
Not sure what that could be. Maybe try to debug into it to identify the issue. I'll try to add devcontainer support soonish, which should help. 

Happy to hear that you made visualization work!

The dataset generation pipeline is quite lengthy and we currently do not plan to publish it. As mentioned in the paper it's Blender based. We render the images and thus, are also able to compute the annotations.

**Comment by a-nau** [comment]
Closed due to inactivity. Also, I'll merge in the CI that runs a few training steps soon (currently on a different branch).

**Comment by NhanDoremon** [comment]
Hi @GauravGajbhiye 

I'm also struggling with problem with you now. I know that it's a long time from your issue posted, but coud you please share your solotion to test and visuallize the images?

Thank you!
